pkg/vm/vm.go

```diff
@@ -377,7 +377,7 @@ func (vm *VirtualMachine) updatePc(
 		}
 
 		// todo(rodro): math check when relAddr is negative
-		relAddr, err := res.Uint64()
+		relAddr, err := op1Cell.Value.Uint64()
```

Comment:
You shouldn't access a Cell Value field, if you want to have it you should use the `Read()` method

Reply:
Also, a bug that was existing previously, is that you cannot extract the value as a uint64 but as a field element

Reply:
I changed it to Read()